applications/portfolioDrafts/deletePortfolioDraft.ts

```diff
@@ -0,0 +1,56 @@
+import { DeleteCommand, DeleteCommandInput } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { dynamodbClient as client } from "./utils/dynamodb";
+import { ErrorCodes } from "./models/Error";
+import { ErrorResponse, ErrorStatusCode, NoContentResponse } from "./utils/response";
+
+const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+
+/**
+ * Deletes a Portfolio Draft
+ *
+ * @param event - The DELETE request from API Gateway
+ */
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+
+  if (!portfolioDraftId) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "PortfolioDraftId must be specified in the URL path" },
+      ErrorStatusCode.BAD_REQUEST
+    );
+  }
+
+  if (event.body) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "HTTP request body must be empty" },
+      ErrorStatusCode.BAD_REQUEST
+    );
+  }
+
+  const params: DeleteCommandInput = {
+    TableName: TABLE_NAME,
+    Key: {
+      id: portfolioDraftId,
+    },
+    ReturnValues: "ALL_OLD",
+  };
+
+  const command = new DeleteCommand(params);
+
+  try {
+    const data = await client.send(command);
+    if (!data.Attributes) {
+      return new ErrorResponse(
+        { code: ErrorCodes.INVALID_INPUT, message: "Portfolio Draft with the given ID does not exist" },
+        ErrorStatusCode.NOT_FOUND
+      );
+    }
+    return new NoContentResponse({ message: "Successful operation" });
+  } catch (err) {
```

Comment:
```suggestion
  } catch (err) {
  console.log("Database error: " + err);
```
We should log the database error, to keep it consistent with the other fns.

Reply:
@zachclark-ccpo I agree that is correct _for now_. I'd really like to see more abstractions around our DB interactions so that we don't have to worry about "manually" logging quite so much. We should see if we can get a ticket in the next sprint to support that.

Reply:
Fixed in https://github.com/dod-ccpo/atat-web-api/pull/40/commits/6a7390025deab72285d0bcc32a9b31cabecb6011